src/main.ts

```diff
@@ -79,7 +78,9 @@ async function run(): Promise<void> {
 async function findAllRepos(
   client: Octokit,
   username: string,
-  organization: string
+  organization: string,
+  OWNER: string,
+  REPO: string
```

Comment:
Please follow the casing of the surrounding code here.
```suggestion
  owner: string,
  repo: string
```

Reply:
removed these extra params it is not required